src/Me/Profile/Profile.js

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from "react";
+
+import UserContext from '../../context/userContext/UserContext';
+import Card from '../components/Card/index';
+
+import { ReactComponent as EmailIcon } from '../../assets/me/icon-email.svg';
+import { ReactComponent as SpokenLanguagesIcon } from '../../assets/me/icon-spokenLanguages.svg';
+import { ReactComponent as CountryIcon } from '../../assets/me/icon-country.svg';
+import { ReactComponent as TitleIcon } from '../../assets/me/icon-title.svg';
+import { ReactComponent as TagsIcon } from '../../assets/me/icon-tags.svg';
+import { ReactComponent as AvailableIcon } from '../../assets/me/icon-available.svg';
+import { ReactComponent as UnavailableIcon } from '../../assets/me/icon-unavailable.svg';
+import { ReactComponent as DescriptionIcon } from '../../assets/me/icon-description.svg';
+
+import "./Profile.css";
```

Comment:
Please don't forget to remove it when you've done to convert to styled-components